### PR TITLE
Common layout and style connect page

### PR DESCRIPTION
**Pull request by jwcooper**
This PR adds a common layout for content type pages. The /connect page is now styled to match the other content pages as well.

### CODE REVIEW

File: _layouts/common.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+---
+
+<div id="libraries-page" class="common-layout">
```
Comment (makermelissa) [inline _layouts/common.html:5]: Did you mean to leave the id=libraries-page" for all pages?

File: _layouts/contributing.html [hunk 2]
```diff
@@ -1,59 +1,57 @@
...
-      <a href="https://learn.adafruit.com/contribute-to-circuitpython-with-git-and-github">Contributing to CircuitPython with Git and Github</a>.
-      You can also find us in the #circuitpython channel on the
-      <a href="https://adafru.it/discord">Adafruit Discord</a>.
-    </p>
-    <p>
-      Have an idea for a new driver or library?
-      <a href="https://github.com/adafruit/circuitpython/issues">File an issue on the CircuitPython repo!</a>
-    </p>
-    <h2>Current Status for {{ site.data.libraries.updated_at | date: "%a, %b %d, %Y" }}</h2>
+<div id="libraries-page">
```
Comment (makermelissa) [inline _layouts/contributing.html:6]: Same thing here.

**Review by makermelissa — CHANGES_REQUESTED** [review]
Looks like you used the libraries-page ID multiple times (being in the layout and specific pages). Multiple duplicate IDs in the same page (because of it being in the common layout too) is something we should avoid. If both are supposed to be there, we should change it to use class instead. Also, maybe change it to something less page specific sounding?

**Comment by jwcooper** [comment]
Need to make a couple more fixes quick.

**Review by tannewt — APPROVED** [review]
Looks good to me. Thank you!

**Review by makermelissa — APPROVED** [review]

**Comment by makermelissa** [comment]
Thank you

**Pull request merged**